app/Models/Signup.php

```diff
@@ -43,7 +55,14 @@ class Signup extends Model
      * @var array
      */
     public static $indexes = [
-        'campaign_id', 'updated_at', 'northstar_id', 'id', 'quantity', 'source', 'referrer_user_id',
+        'campaign_id',
+        'id',
+        'group_id',
+        'quantity',
+        'northstar_id',
+        'referrer_user_id',
+        'source',
+        'updated_at',
```

Comment:
Out of scope for this PR, but it seems very odd to me that we'd filter by `quantity` or `updated_at`. I wonder if these are relics from when https://github.com/dosomething/domino was a thing? I do see that there are indexes created in the SQL table though, so... 🤷